src/components/scenes/newAccount/NewAccountReviewScene.tsx

```diff
@@ -40,9 +40,25 @@ const AccountReviewComponent = (props: Props) => {
   const { username, password, pin, onNext } = props
   const theme = useTheme()
   const styles = getStyles(theme)
+  const dispatch = useDispatch()
 
   return (
-    <ThemedScene title={lstrings.review}>
+    <ThemedScene
```

Comment:
This scene can't go back. By the time we get here, the account has been created so username and password can't be changed.

Reply:
Missed removal of some testing code